docs/articles/nunit/getting-started/dotnet-core-and-dotnet-standard.md

```diff
@@ -2,20 +2,21 @@
 
 More information and getting started tutorials are available for NUnit and .NET Core targeting [C#](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-with-nunit), [F#](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-fsharp-with-nunit) and [Visual Basic](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-visual-basic-with-nunit) in the .NET Core documentation's [Unit Testing in .NET Core and .NET Standard](https://docs.microsoft.com/en-us/dotnet/core/testing/) page.
 
-Each test project needs to reference version 4.1.0 or later of the NUnit3 Visual Studio Test Adapter.
+The other information on this page is older documentation. If you follow the instructions in the [Installation section](xref:installation) your project will work with .NET Core and .NET Standard.
 
-It is recommended to install the adapter from NuGet if you are testing .NET Core or .NET Standard projects.
-The VSIX adapter has been deprecated for VS2019. Latest version is 3.17.0. For VS2022 there is no VSIX adapter.
+The test projects have to be .Net Core or .Net Framework, .Net Standard can't be used as a test project, since it can't be run on its own, but any code in a .Net Standard library can be tested from a .Net Core or .Net Framework test project.
```

Comment:
```suggestion
The test projects have to be .NET Core or .NET Framework; .NET Standard can't be used as a test project, since it can't be run on its own, but any code in a .NET Standard library can be tested from a .NET Core or .NET Framework test project.
```